src/phoenix/trace/openai/instrumentor.py

```diff
@@ -290,7 +297,9 @@ def __next__(self) -> ChatCompletionChunk:
                     LLM_OUTPUT_MESSAGES: _accumulate_messages(
                         chunks=self._self_chunks, num_choices=self._self_context.num_choices
                     ),  # type: ignore
-                    OUTPUT_VALUE: json.dumps([chunk.json() for chunk in self._self_chunks]),
+                    OUTPUT_VALUE: json.dumps(
+                        {"chat_completion_chunks": [chunk.dict() for chunk in self._self_chunks]}
```

Comment:
Do we absolutely need this key? It seems to be too much "cloak-and-dagger".

Reply:
addressed in [c5213ba](https://github.com/Arize-ai/phoenix/pull/1888/commits/c5213ba169a27b45d4903a1e78da6eb63c764719)